pom.xml

```diff
@@ -28,7 +28,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.520</version>
+    <version>1.609</version>
```

Comment:
this isn't actually a plugin - it's a submodule that is a plugin - so would be a bit more work, but...
given the severity of the issue (this causes credentials not to load which causes so much other stuff not to load that you can;t even start Jenkis) I wanted to do as small a change as possible (so it would be as uncontroversial as possible).